Autopilot/Inc/imu.hpp

```diff
@@ -84,4 +84,6 @@ class BMX160: public IMU{
 
 };
 
+void ImuTxRxCallback();
```

Comment:
Add this above this prototype pls:

        /**
         * Callback for exclusive use by the SPI ISR.
         * Users of this module should never call this function.
         * Any manipulation of this function by the higher level voids all guarantees made by this module.
         * */